README.md

```diff
@@ -105,7 +96,7 @@ root_cert = None
 for certificate in cert_authorities:
     verifier = VerifierBuilder().add_root_certificate(certificate).build()
     try:
-        verifier.verify(timestamp_response, message_hash)
+        verifier.verify_message(timestamp_response, message)
```

Comment:
Keep the original example that accepts a pre-hashed message, in case folks are using other hash algorithms that this library doesn't directly support.

Reply:
I feel like the example in the README should show the intended golden path (which I think the new method is)... Of course it would be nice to have API docs so the options were visible there.